src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,218 @@
+        }
+    }
+
+    private void insertDriversToCarsDrivers(Car car) {
+        String insertDriversQuery = "INSERT INTO cars_drivers (car_id, driver_id) "
+                + "VALUES (?, ?);";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement insertDriversStatement
+                         = connection.prepareStatement(insertDriversQuery)) {
+            insertDriversStatement.setLong(1, car.getId());
+            for (Driver driver : car.getDrivers()) {
+                insertDriversStatement.setLong(2, driver.getId());
+                insertDriversStatement.executeUpdate();
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't create car"
+                    + car + ". ", e);
+        }
+    }
+
+    private List<Driver> getDriversForCar(Long carId) {
+        String getListDriversQuery = "SELECT id, name, licenceNumber "
+                + "FROM drivers "
+                + "JOIN cars_drivers "
+                + "ON drivers.id = cars_drivers.driver_id "
+                + "WHERE cars_drivers.car_id = ? AND drivers.is_deleted = FALSE "
+                + "AND cars_drivers.is_deleted = FALSE;";
+        List<Driver> drivers = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement statement = connection.prepareStatement(getListDriversQuery)) {
+            statement.setLong(1, carId);
+            ResultSet resultSet = statement.executeQuery();
+            while (resultSet.next()) {
+                drivers.add(getDriversFromResultSet(resultSet));
+            }
+            return drivers;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't get a list of cars from carsDB.", e);
+        }
+    }
+
+    private Driver getDriversFromResultSet(ResultSet resultSet) throws SQLException {
+        Driver driver = new Driver();
+        driver.setId(resultSet.getObject("id", Long.class));
+        driver.setName(resultSet.getString("name"));
+        driver.setLicenseNumber(resultSet.getString("licenceNumber"));
+        return driver;
+    }
+
+    private Car getCarFromStatement(ResultSet resultSet) throws SQLException {
+        Car car = new Car();
+        Long id = resultSet.getObject("id", Long.class);
+        String model = resultSet.getString("model");
+        car.setId(id);
+        car.setModel(model);
+        Manufacturer manufacturer = new Manufacturer();
+        manufacturer.setId(resultSet.getObject("manufacturer_id", Long.class));
+        manufacturer.setName(resultSet.getString("name"));
+        manufacturer.setCountry(resultSet.getString("country"));
+        car.setManufacturer(manufacturer);
+        return car;
+    }
+
+    private void removeOldConnectionsInCarsDrivers(Car car) {
+        String removeConnectionsQuery = "UPDATE cars_drivers SET is_deleted = TRUE "
```

Comment:
do you have `is_deleted` column in the `cars_drivers ` table? If you have `is_deleted`, add this column to `init_db.sql`, if you don't have `is_deleted` then you can delete rows from `cars_drivers ` without soft delete